geomesa-hbase/geomesa-hbase-spark/src/main/scala/org/locationtech/geomesa/spark/hbase/GeoMesaHBaseInputFormat.scala

```diff
@@ -42,6 +43,17 @@ class GeoMesaHBaseInputFormat extends InputFormat[Text, SimpleFeature] with Lazy
     delegate.setConf(conf)
     // see TableMapReduceUtil.java
     HBaseConfiguration.merge(conf, HBaseConfiguration.create(conf))
+    val auth = conf.get("hbase.security.authentication")
+    auth match{
+      case "kerberos" => {
+        conf.set("hadoop.security.authentication", "Kerberos")
```

Comment:
From some testing, I think it might be better to call 'SecurityUtil.setAuthenticationMethod(AuthenticationMethod.KERBEROS, conf)' here.  

After the configuration is built up, I think it might be worth a quick call to 'HBaseAdmin.checkHBaseAvailable(conf)'.  If this method fails, we'd know immediately if the Kerberos bits are not aligned.